### ISSUE TITLE
Remove leading digit from phone number

### ISSUE THREAD

**Issue by 3m-tk** [CB954/SkypeToTeams#3]
Hi All,

I have phone numbers stored within Connectwise Manage which are not compliant with E.164 standards (eg. 0411 111 111 and not as +61 411 111 111). 

I have tried to amend the script to strip the leading character from the phone number without success (I have changed the digit within **telUri = `"tel:`" & Right(telUri,Len(telUri)-6) 'REMOVE PRECEEDING skype: STRING, ADD PREFIX tel:** from 6 to 7 and this doesn't seem to work either). 

I have also tried to add a dial plan within the Teams admin centre to strip the leading zero but again without success. 

![image](https://user-images.githubusercontent.com/97484720/161423771-776cd846-f2fd-40fd-8cac-ad0677429b3b.png)

Can anyone point me in the right direction on how to approach this? 

Any input would be greatly appreciated.

**Comment by XAte6** [comment]
Hi @3m-tk

You don't seem to be getting a response from the original author, so I'll try to help.

I've re-written this as a VBScript instead of the batch file, which means the black command prompt window doesn't flash up. I also changed it so that the url type tel:xxxxx was for the whole machine rather than just the current user. Unfortunately, that required Administrator access to the Powershell command prompt for install.

If you want a single user only version, let me know and I'll adjust.

Basically my fix will remove the "skype:" prefix and the trailing "?call". Then if the number starts with "0", it'll be replaced with "+61" and then "tel:" added as the prefix.

I've attached a fix for you to try here:
[install.ps1.txt](https://github.com/CB954/SkypeToTeams/files/8466031/install.ps1.txt)
